examples/offline_inference/basic/async.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import random
+import uuid
+
+import ray
+
+from vllm import AsyncEngineArgs, AsyncLLMEngine, SamplingParams
+
+
+def generate_prompt():
+    prompts = [
+        "Hello, my name is",
+        "The president of the United States is",
+        "The capital of France is",
+        "The future of AI is",
+    ]
+    return random.choice(prompts)
+
+
+@ray.remote(num_gpus=1)
+class InferenceEngine:
+
+    def __init__(self, **kwargs):
+        self.args = AsyncEngineArgs(**kwargs)
+        self.engine = AsyncLLMEngine.from_engine_args(self.args)
```

Comment:
The v0 `AsyncLLMEngine` is now deprecated, could you change this to use `AsyncLLM`?

Reply:
I think it `AsyncLLMEngine` refer to  v1`AsyncLLM` is v1 is enabled since https://github.com/vllm-project/vllm/blob/main/vllm/engine/async_llm_engine.py#L1239-L1242
```python
if envs.is_set("VLLM_USE_V1") and envs.VLLM_USE_V1:
    from vllm.v1.engine.async_llm import AsyncLLM

    AsyncLLMEngine = AsyncLLM  # type: ignore
```


After the stable version of v1 engine is released, we should link the `AsyncLLMEngine` defined in https://github.com/vllm-project/vllm/blob/main/vllm/__init__.py#L48 the v1 `AsyncLLM` without change the user side API, right ?